products/quarkus.md

```diff
@@ -7,7 +7,7 @@ permalink: /quarkus-framework
 alternate_urls:
 -   /quarkus
 releasePolicyLink: https://quarkus.io/security/
-changelogTemplate: "https://github.com/quarkusio/quarkus/releases/tag/__LATEST__.Final"
+changelogTemplate: "https://github.com/quarkusio/quarkus/releases/tag/__LATEST__"
```

Comment:
This change is not related to Chrome OS and must be reverted.

FYI this Quarkus change has been taken into account in #3467.